react/components/ActionMenu/README.md

```diff
@@ -0,0 +1,244 @@
+#### Action Menus are contextual menus that let the user chose and execute actions from a list.
+
+### 👍 Dos
+- Use _isDangerous_ for destructive actions that delete data or lead to a state that is hard to recover. Danger buttons should always have a confirmation dialog.
+
+### 👎 Don'ts
+- It's OK to hide the caret when using the OptionsDots icon, but avoid otherwise. It's important to signify that this button behaves differently than normal ones.
```

Comment:
It's important to notice that this button behaves differently than the regular.

Reply:
@salesfelipe acho que era pra ser `signify` mesmo. O caret serve como um sinal visual de que este botão é diferente do normal. Um desenvolvedor escondendo esse caret deve lembrar disso.